ClangSharpPInvokeGenerator/Cursors/Attrs/Attr.cs

```diff
@@ -5,6 +5,54 @@ namespace ClangSharpPInvokeGenerator
 {
     internal class Attr : Cursor
     {
+        public static new Attr Create(CXCursor handle, Cursor parent)
```

Comment:
The `Cursor.Create` method was split out into various sub `Create` methods so that we can try to return the "best" base type for anything we don't currently recognize.